### ISSUE TITLE
Can not train a  good-performance model

### ISSUE THREAD

**Issue by ZyUestc** [f90/Wave-U-Net-Pytorch#2]
Could you share a pre-trained model? I train several times and still not get a applicable model, or should I assign a random seed? Many thanks!

**Comment by f90** [comment]
Are you using the MUSDB dataset for training and the default training parameters? I don't think this is seed-related. What do you mean with applicable model? Does the training loss not converge properly at all? Or does it just not perform as well as you had hoped? 

Pretrained models are available already, see the Readme file!

**Comment by zrq-7** [comment]
Hi,I can not open the pre-trained model，Web page cannot open!Can you share it again？very thanks.

**Comment by f90** [comment]
@zrq-7 The link here still works for me:
https://www.dropbox.com/s/r374hce896g4xlj/models.7z?dl=1
